### DISCUSSION TITLE
Clicking on mods while watching a replay will enable/disable them.

### DISCUSSION THREAD

**Discussion by OIAGYT** [ppy/osu#23162] [Ideas] [answered]
This is already in osu!stable but it's a nice feature to have so why not have it in lazer too?

**Comment by nagi-desuuu** [comment, marked as answer]
https://github.com/ppy/osu/issues/8357

**Reply by OIAGYT** [reply]
> #8357

sorry, i wasn't able to find it despite me looking for it.
